AwsCryptographyPrimitives/runtimes/python/src/aws_cryptography_primitives/smithygenerated/aws_cryptography_primitives/client.py

```diff
@@ -0,0 +1,620 @@
+                InterceptorContext[Input, None, DafnyRequest, DafnyResponse], context
+            )
+
+            context_with_response._transport_response = config.dafnyImplInterface.handle_request(
+                input=context_with_response.transport_request
+            )
+
+            # Step 7n: Invoke read_after_transmit
+            for interceptor in interceptors:
+                interceptor.read_after_transmit(context_with_response)
+
+            # Step 7o: Invoke modify_before_deserialization
+            for interceptor in interceptors:
+                context_with_response._transport_response = (
+                    interceptor.modify_before_deserialization(context_with_response)
+                )
+
+            # Step 7p: Invoke read_before_deserialization
+            for interceptor in interceptors:
+                interceptor.read_before_deserialization(context_with_response)
+
+            # Step 7q: deserialize
+            context_with_output = cast(
+                InterceptorContext[Input, Output, DafnyRequest, DafnyResponse],
+                context_with_response,
+            )
+            context_with_output._response = await deserialize(
+                context_with_output._transport_response, config
+            )
+
+            # Step 7r: Invoke read_after_deserialization
+            for interceptor in interceptors:
+                interceptor.read_after_deserialization(context_with_output)
+        except Exception as e:
+            if context.response is not None:
+                # config.logger.exception(f"Exception occurred while handling: {context.response}")
+                pass
+            context._response = e
+
+        # At this point, the context's request and transport_request have definitively been set,
+        # the response is either set or an exception, and the transport_resposne is either set or
+        # None. This will also be true after _finalize_attempt because there is no opportunity
+        # there to set the transport_response.
+        attempt_context = cast(
+            InterceptorContext[Input, Output, DafnyRequest, DafnyResponse | None], context
+        )
+        return await self._finalize_attempt(interceptors, attempt_context)
+
+    async def _finalize_attempt(
+        self,
+        interceptors: list[Interceptor[Input, Output, DafnyRequest, DafnyResponse]],
+        context: InterceptorContext[Input, Output, DafnyRequest, DafnyResponse | None],
+    ) -> InterceptorContext[Input, Output, DafnyRequest, DafnyResponse | None]:
+        # Step 7s: Invoke modify_before_attempt_completion
+        try:
+            for interceptor in interceptors:
+                context._response = interceptor.modify_before_attempt_completion(
+                    context
+                )
+        except Exception as e:
+            if context.response is not None:
+                # config.logger.exception(f"Exception occurred while handling: {context.response}")
```

Comment:
nit: commented out code

Reply:
Interestingly, this is actually from Smithy-Python. I will override their codegen.

Reply:
oh, fun times, not a huge deal either way if it's codegen'd, i assumed it wasn't